### PR TITLE
Update TS to latest release

### PR DESCRIPTION
**Pull request by nadyaA**
Fix corresponding tests

https://github.com/telerik/mobile-cli-lib/pull/980

### CODE REVIEW

File: lib/services/ios-debug-service.ts [hunk 1]
```diff
@@ -171,20 +171,20 @@ class IOSDebugService extends DebugServiceBase implements IPlatformDebugService
...
 	}
 
-	private async deviceStart(debugData: IDebugData, debugOptions: IDebugOptions): Promise<string[]> {
+	private async deviceStart(debugData: IDebugData, debugOptions: IDebugOptions): Promise<string> {
 		await this.$devicesService.initialize({ platform: this.platform, deviceId: debugData.deviceIdentifier });
 		const action = async (device: Mobile.IiOSDevice) => device.isEmulator ? await this.emulatorStart(debugData, debugOptions) : await this.deviceStartCore(device, debugData, debugOptions);
-		const results = await this.$devicesService.execute(action, this.getCanExecuteAction(debugData.deviceIdentifier));
-		return _.map(results, r => r.result);
+		const deviceActionResult = await this.$devicesService.execute(action, this.getCanExecuteAction(debugData.deviceIdentifier));
+		return deviceActionResult[0].result;
```
Comment (rosen-vladimirov) [inline lib/services/ios-debug-service.ts]: previously this has not been failing when the `deviceActionResult` is `null/undefined` or empty array, but now it will fail with error - `Cannot read property result of undefined.` Is this expected? In case not, you can use:
```TypeScript
return deviceActionResult && deviceActionResult[0] && deviceActionResult[0].result;
```

File: test/plugins-service.ts [hunk 2]
```diff
@@ -403,17 +391,15 @@ describe("Plugins service", () => {
 						}
 					},
 					"devDependencies": {
-						"grunt": "0.4.2"
+						"grunt": "0.4.`2`"
```
Comment (rosen-vladimirov) [inline test/plugins-service.ts]: ?

File: lib/services/livesync/debug-livesync-service.ts [hunk 3]
```diff
@@ -66,7 +66,7 @@ export class DebugLiveSyncService extends LiveSyncService implements IDebugLiveS
 		const buildConfig: IBuildConfig = _.merge({ buildForDevice: !deviceAppData.device.isEmulator }, deployOptions);
 		debugData.pathToAppPackage = this.$platformService.lastOutputPath(debugService.platform, buildConfig, projectData);
 
-		this.printDebugInformation(await debugService.debug<string[]>(debugData, debugOptions));
+		this.printDebugInformation([await debugService.debug(debugData, debugOptions)]);
```
Comment (rosen-vladimirov) [inline lib/services/livesync/debug-livesync-service.ts]: we always call `printDebugInformation` by manually converting some string to array, can't we change the `printDebugInformation` method to has `string` for parameter instead of `string[]`

File: lib/services/ios-debug-service.ts (hunk 1 above)
Comment (rosen-vladimirov) [inline lib/services/ios-debug-service.ts]: Agreed that this will never happen. Disregard the comment,

File: lib/commands/debug.ts [hunk 4]
```diff
@@ -22,16 +22,16 @@
 		let debugData = this.$debugDataService.createDebugData(this.$projectData, this.$options);
 
 		await this.$platformService.trackProjectType(this.$projectData);
+		const devices = this.$devicesService.getDeviceInstances();
```
Comment (rosen-vladimirov) [inline lib/commands/debug.ts]: why is this line moved? By definition the `const`s should be declared where they are used, not before that.

**Comment by nadyaA** [comment]
run ci
